src/components/Viewer/Footer/DownloadButton.jsx

```diff
@@ -28,7 +28,7 @@ const DownloadButton = ({ file, t }) => {
       extension="full"
       theme="secondary"
       icon={DownloadIcon}
-      label={t('Viewer.actions.download')}
+      label={t('viewer.actions.download')}
       onClick={() => handleClick(file)}
     />
   )
```

Comment:
ah ok les changements sont dans le commit d'après... qui s'appelle `feat: Add withLocales to ForwardButton` mais qui ne fait pas que ça. On pourrait peut-être rassembler les sujets ?

Reply:
Oui en effet, je pense que ça sera plus compréhensible.. :)